### PR TITLE
Version match fix

### PR DESCRIPTION
**Pull request by erithmetic**

### CODE REVIEW

File: lib/geocode_records/smarty_streets.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    def self.compatible?
+      output = run('-V')
+      current_version = Gem::Version.new output.chomp
+      base_version = Gem::Version.new '1.3.2'
+      current_version >= base_version
+    end
+
+    def self.run(*args)
+      shargs = Shellwords.join(args)
+      `#{bin_path} #{shargs}`
```
Comment (seamusabshere) [inline lib/geocode_records/smarty_streets.rb:27]: @dkastner this should be implemented with system so that output is passed through, etc.

Comment (erithmetic) [inline lib/geocode_records/smarty_streets.rb:27]: When I used system I got back `true` so the backticks actually gave me output

Comment (seamusabshere) [inline lib/geocode_records/smarty_streets.rb:27]: @dkastner right - you can use backticks for the -V, but system for the real geocode

**Comment by erithmetic** [comment]
@seamusabshere the whole test suite works fine with backticks

**Comment by seamusabshere** [comment]
@dkastner smartystreets generates a lot of useful output - catching it all in a (huge) string is not desired behavior

`Kernel.system` is preferred

**Pull request merged**